packages/react/src/generators/library/lib/add-rollup-build-target.ts

```diff
@@ -69,38 +69,38 @@ export async function addRollupBuildTarget(
   if (hasRollupPlugin) {
     // New behavior, using rollup config file and inferred target.
     host.write(
-      joinPathFragments(options.projectRoot, 'rollup.config.js'),
-      stripIndents`
-      const { withNx } = require('@nx/rollup/with-nx');
-      const url = require('@rollup/plugin-url');
-      const svg = require('@svgr/rollup');
-      
-      module.exports = withNx({
-        main: '${maybeJs(options, './src/index.ts')}',
-        outputPath: '${joinPathFragments(
-          offsetFromRoot(options.projectRoot),
-          'dist',
-          options.projectRoot
-        )}',
-        tsConfig: './tsconfig.lib.json',
-        compiler: '${options.compiler ?? 'babel'}',
-        external: ${JSON.stringify(external)},
-        format: ['esm'],
-        assets:[{ input: '.', output: '.', glob: 'README.md'}],
-      }, {
-        // Provide additional rollup configuration here. See: https://rollupjs.org/configuration-options
-        plugins: [
-          svg({
-            svgo: false,
-            titleProp: true,
-            ref: true,
-          }),
-          url({
-            limit: 10000, // 10kB
-          }),
-        ],
-      });
-    `
+      joinPathFragments(options.projectRoot, 'rollup.config.cjs'),
+      `
+const { withNx } = require('@nx/rollup/with-nx');
+const url = require('@rollup/plugin-url');
+const svg = require('@svgr/rollup');
+
+module.exports = withNx({
+  main: '${maybeJs(options, './src/index.ts')}',
+  outputPath: '${joinPathFragments(
+    offsetFromRoot(options.projectRoot),
+    'dist',
+    options.projectRoot
+  )}',
+  tsConfig: './tsconfig.lib.json',
+  compiler: '${options.compiler ?? 'babel'}',
+  external: ${JSON.stringify(external)},
+  format: ['esm'],
+  assets:[{ input: '.', output: '.', glob: 'README.md'}],
+}, {
```

Comment:
The indentation doesn't look correct here. This line (90) has the closing curly braces with no indentation.

Reply:
Ah, yeah if it's run through Prettier. I guess my default style doesn't fit. 😓